code/memanalyzer.py

```diff
@@ -92,24 +93,26 @@ def analyse_memory(elf_file):
         # print("{0: >10}|{1: >30}|{2:12X}|{3:12X}|{4:8}".format(id_, descr, section_start, section_end, section_length))
         # i += 1
 
-    # print("Total Used RAM : %d" % usedRAM)
-    # print("Free RAM : %d" % (TOTAL_DRAM - usedRAM))
-    # print("Free IRam : %d" % usedIRAM)
+    # print("Total Used RAM : {:d}".format(usedRAM))
+    # print("Free RAM : {:d}".format(TOTAL_DRAM - usedRAM))
+    # print("Free IRam : {:d}".format(usedIRAM))
     return ret
 
 
 def run(env_, modules_):
     flags = ""
-    for item in modules_.items():
-        flags += "-D%s_SUPPORT=%d " % item
-    command = "ESPURNA_BOARD=\"WEMOS_D1_MINI_RELAYSHIELD\" ESPURNA_FLAGS=\"%s\" platformio run --silent --environment %s 2>/dev/null" % (flags, env_)
+    for k, v in modules_.items():
+        flags += "-D{}_SUPPORT={:d} ".format(k, v)
```

Comment:
This was doing auto tuple unpacking.   I made it more explicit.    This took some time to decipher.  :)